CRM/Core/Payment/PayPalImpl.php

```diff
@@ -687,9 +687,20 @@ public function cancelSubscription(&$message = '', $params = array()) {
    * We don't need to handle paypal standard using this path as there has never been any historic support
    * for paypal standard to call civicrm/payment/ipn as a path.
    */
-  static public function handlePaymentNotification() {
-    $paypalIPN = new CRM_Core_Payment_PayPalProIPN($_REQUEST);
```

Comment:
NB - the code that handles the incoming is here https://github.com/davecivicrm/civicrm-core/blob/e4555ff1269c3e8034375cb9877daca298001164/CRM/Core/Payment.php#L1140-L1140